config/jobs/kubernetes-sigs/controller-runtime/controller-runtime-periodics-master.yaml

```diff
@@ -2,18 +2,22 @@ periodics:
 - interval: 6h
   name: periodic-controller-runtime-test-master
   decorate: true
+  labels:
+      preset-service-account: "true"
   extra_refs:
   - org: kubernetes-sigs
     repo: controller-runtime
     base_ref: master
   spec:
     containers:
-    - image: golang:1.13
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20200720-4766100-1.18
```

Comment:
why kubekind, why service account?

Reply:
I matched the same that we're using on the Cluster API side, I forget why the service account was needed, although I could look tomorrow if necessary (logs or something?). The main reason that `kubekins` is nice is that the images are hosted on GCR, and the embedded go version is the same that Kubernetes has been built with. For example `1.18` here uses Go 1.13. When in the future we change the built-in dependencies to 1.19, we should build with Go 1.15, the images would already have it.

Reply:
I don't think we need the service account for this project, will remove it.

Reply:
Yeah but we pull a gigantic image we don't need and use wrapping that makes reproducing everything harder. I think updating the go minor version on the job when we update the kube dependencies shouldn't be too difficult

Reply:
Fixed